code/game/objects/items/weaponry.dm

```diff
@@ -237,6 +237,11 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	resistance_flags = FIRE_PROOF
 	total_mass = TOTAL_MASS_MEDIEVAL_WEAPON
 
+/obj/item/katana/lavaland
+	desc = "Woefully underpowered in Lavaland."
```

Comment:
the true design of mining: all of the mining gear SUCKS on mining and is OP on station

Reply:
I thought it was a good joke tbh, its p-good on station imo

Reply:
yes that's why the description is funny